MdeModulePkg/Test/Library/MockBaseResetSystemLibNull/MockBaseResetSystemLibNull.c

```diff
@@ -0,0 +1,37 @@
+/** @file
+  Mocked version of ResetSystemLib for SetupDataPkg unit tests
+  Copyright (c) 2006 - 2019, Intel Corporation. All rights reserved.<BR>

```

Comment:
if these are new files, let's update the SPDX to have Microsoft's copyright.

Reply:
The file isn't new, the location is. Does that warrant changing the copyright?

Reply:
No, if it is just copied, we can leave it.